app/src/ui/lib/app-proxy.ts

```diff
@@ -60,14 +79,13 @@ export function getUserDataPath(): string {
  *
  * This is preferable to using `remote` directly because we cache the result.
  */
-export function getDocumentsPath(): string {
+export async function getDocumentsPath(): Promise<string> {
   if (!documentsPath) {
-    const app = getApp()
```

Comment:
Now that #13803 is merged, I think you can also remove the `getApp` function entirely, right?

Reply:
Yep! That was my plan - not sure it would have even built after merging in dev.